docs/en_us/platform_api/source/courses/overview.rst

```diff
@@ -0,0 +1,64 @@
+.. _Courses API Overview:
+
+#############################
+Courses API Overview
+#############################
+
+Use the Courses API to obtain information about edX courses.
+
+.. contents::
+   :local:
+   :depth: 1
+
+*****************************************
+Courses API Version and Status
+*****************************************
+
+The Courses API is currently at version 1.0.
```

Comment:
Oh, I meant that as a minor wording edit. I have no reason to believe it's not v1.0. Sorry!